feature/community/src/main/java/com/iteneum/community/domain/CommunitySectionList.kt

```diff
@@ -0,0 +1,11 @@
+package com.iteneum.community.domain
+
+/**
+ * This class represent a filther and support for the CommunityCardDataModel
+ * in order to be able to access strings resources and set propper name to the cardButton in UI Community
+ *
+ * @author Andrés Ivan Medina Herrera
+ */
+enum class CommunitySectionList {
```

Comment:
Something like this would be a similar approach but with sealed classes and there's no need for acceptance criteria, it is just a different programming style using kotlin features.


![Screenshot 2023-04-20 at 7 03 50 PM](https://user-images.githubusercontent.com/105077839/233746353-ec1cf20e-3f14-49cc-839d-c134fa4be4f5.png)

![Screenshot 2023-04-20 at 7 03 35 PM](https://user-images.githubusercontent.com/105077839/233746358-8a1151b7-4408-41b8-9c27-f668e1df030c.png)

Reply:
the acceptance criteria is because a different component as a sealed class will imply new component decisions and connections  over the objects that will be handled as options for the sealed class, this decisions concern to the Project master and the architect so as much as I know @irv205  or @dmroldanch  should take the decision  but on previous talk  

they mentioned this so : check with them @melissaquezada94 
![imagen](https://user-images.githubusercontent.com/128720218/234021824-d8a6b38d-f370-4130-9da0-d68637148962.png)